firmware/lib/TestStandCommController/src/TestStandCommController.cxx

```diff
@@ -41,11 +42,15 @@ SerialResult TestStandCommController::status(Status status)
     return this->session.send_message(msg);
 }
 
-SerialResult TestStandCommController::data(uint8_t *data, uint8_t length)
+SerialResult TestStandCommController::position(int32_t x, int32_t y)
```

Comment:
Should the type be `uint32_t`? are we expecting negative values?

Reply:
Yes, the position is allowed to be negative (I made this change in the motor_refactor PR). There's not really any reason to limit it (it would just create possible wraparound bugs that would be hard to track down)

Reply:
Is this position specified in counts?

Reply:
I assume all distances in firmware are in counts or steps, right?

Reply:
Yes counts